literature.bib

```diff
@@ -8681,25 +8684,25 @@ @inproceedings{KLG:MSR11
 } 
 
 @article{CMN:STVR11,
- 	 author={Cartaxo, Emanuela G and Machado, Patr{\'\i}cia DL and Neto, Francisco G Oliveira},
- 	 title={{On the Use of a Similarity Function for Test Case Selection in the Context of Model-Based Testing}},
- 	 journal=STVR,
-  	volume={21},
-  	number={2},
-  	pages={75--100},
- 	 publisher={Wiley Online Library},
- 	 year=2011
+ 	 author = {Cartaxo, Emanuela G and Machado, Patr{\'\i}cia DL and Neto, Francisco G Oliveira},
+ 	 title = {{On the Use of a Similarity Function for Test Case Selection in the Context of Model-Based Testing}},
+ 	 journal = STVR,
+  	volume = {21},
+  	number = {2},
+  	pages = {75--100},
+ 	 publisher = {Wiley Online Library},
```

Comment:
Isnt this already a variable in MYabrv?

Reply:
This is neither of the Wiley organs present in MYabrv.